modules/EnsEMBL/Web/ImageConfig/gene_summary.pm

```diff
@@ -55,7 +55,7 @@ sub init_cacheable {
   $self->add_tracks('other',
     [ 'scalebar',  '', 'scalebar',  { display => 'normal', strand => 'b', name => 'Scale bar', description => 'Shows the scalebar' }],
     [ 'ruler',     '', 'ruler',     { display => 'normal', strand => 'b', name => 'Ruler',     description => 'Shows the length of the region being displayed' }],
-    [ 'draggable', '', 'draggable', { display => 'normal', strand => 'b', menu => 'no' }],
+    [ 'draggable', '', 'draggable', { display => 'normal', strand => 'r', menu => 'no' }],
```

Comment:
As @ens-ap5 suggested, probably best to remove the line completely and see if it still works. I do have a doubt that it gets added by default for all but do check if removing the line does as expected.